Drop-In/src/main/java/com/braintreepayments/api/DropInEvent.java

```diff
@@ -0,0 +1,107 @@
+package com.braintreepayments.api;
+
+import android.os.Bundle;
+import android.os.Parcel;
+import android.os.Parcelable;
+
+class DropInEvent {
```

Comment:
This looks awesome - much easier to read / understand than the separate event classes prior

Reply:
🙏 ! It can still evolve too. The closer we get to GA I think we can clear up even more.